api/v1/swagger.yaml

```diff
@@ -41,8 +42,8 @@ basePath: /v1
 
 # =============================================================================
 schemes:
-  - https
   - http
+  - https
```

Comment:
@BrandonMurnane curious to know why this change? Does this change the "try it out feature" to default to http instead of https? Would this have any side effects in terms of api working?